SQLQuery2.sql

```diff
@@ -0,0 +1,13 @@
+create table tblEnemy(EnemyId Int PRIMARY KEY,EnemyName varchar(255),Description varchar(255))
+create table tblEpisodeEnemy(EpisodeEnemyId Int PRIMARY KEY,EpisodeId Int,EnemyId Int, FOREIGN KEY(EnemyId)REFERENCES tblEnemy(EnemyId))
+create table tblAuthor(AuthorId Int PRIMARY KEY,AuthorName varchar(255))
```

Comment:
you do not have to prefix each table name with "tbl"

Reply:
i will fix it next time 👌